x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -0,0 +1,350 @@
+                  <EuiSelect
+                    options={getIntervalOptions(bucketSpan)}
+                    value={interval}
+                    onChange={(e) => setInterval(e.target.value)}
+                    aria-label={i18n.translate(
+                      'xpack.ml.jobsList.datafeedModal.intervalSelection',
+                      {
+                        defaultMessage: 'Datafeed modal chart interval selection',
+                      }
+                    )}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EditQueryDelay datafeedConfig={datafeedConfig} />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiFlexGroup gutterSize="none" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty
+                    color="primary"
+                    size="l"
+                    onClick={() => {
+                      handleEndDateChange(CHART_DIRECTION.BACK);
+                    }}
+                    iconType="arrowLeft"
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <Chart size={CHART_SIZE}>
+                    <Settings showLegend showLegendExtra legendPosition={Position.Bottom} />
+                    <Axis
+                      id="bottom"
+                      position={Position.Bottom}
+                      showOverlappingTicks
+                      tickFormat={dateFormatter}
+                      title={i18n.translate('xpack.ml.jobsList.datafeedModal.xAxisTitle', {
+                        defaultMessage: 'Bucket span ({bucketSpan})',
+                        values: { bucketSpan },
+                      })}
+                    />
+                    <Axis
+                      id="left"
+                      title={i18n.translate('xpack.ml.jobsList.datafeedModal.yAxisTitle', {
+                        defaultMessage: 'Record count',
+                      })}
+                      position={Position.Left}
+                    />
+                    <LineSeries
+                      color={euiTheme.euiColorPrimary}
+                      id={i18n.translate('xpack.ml.jobsList.datafeedModal.sourceSeriesId', {
+                        defaultMessage: 'Source doc count by bucket span',
+                      })}
+                      xScaleType={ScaleType.Time}
+                      yScaleType={ScaleType.Linear}
+                      xAccessor={0}
+                      yAccessors={[1]}
+                      data={sourceData}
+                      curve={CurveType.LINEAR}
+                    />
+                    <LineSeries
+                      color={euiTheme.euiColorAccentText}
+                      id={i18n.translate('xpack.ml.jobsList.datafeedModal.bucketSeriesId', {
+                        defaultMessage: 'Doc count per bucket',
```

Comment:
Likewise, IMO the important detail here is where this info is coming from and we can avoid duplicating the chart axis labels. For example:
```suggestion
                        defaultMessage: 'Job results',
```

Reply:
Updated in b201a635f08667349e20b1631023c46d1162a3b5